### PR TITLE
Whiteboard display name is updated now

### PR DESCRIPTION
**Pull request by hero101**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/domain/common/whiteboard/whiteboard.service.ts [hunk 1]
```diff
@@ -126,24 +126,23 @@ export class WhiteboardService {
...
         updateWhiteboardData.profileData
       );
     }
-    return await this.save(whiteboard);
+    return this.save(whiteboard);
   }
 
   async save(whiteboard: IWhiteboard): Promise<IWhiteboard> {
-    return await this.whiteboardRepository.save(whiteboard);
+    return this.whiteboardRepository.save(whiteboard);
```
Comment (alstojanovic) [inline src/domain/common/whiteboard/whiteboard.service.ts]: I don't think removing this await changes anything. Why did you remove it?

Comment (hero101) [reply src/domain/common/whiteboard/whiteboard.service.ts]: It's not required for the code to execute. The function gives the impression that it's doing something but actually just returning a result.

**Review by alstojanovic — APPROVED** [review]
